index.html

```diff
@@ -50,126 +59,132 @@ <h2>I'm a Software Developer</h2>
+                        <div class="worksListCard">
+                            <ul class="aligned-list-card">
+                                <li>Html</li>
+                                <li>CSS</li>
+                                <li>JavaScript</li>
+                            </ul>
+                        </div>
+                        <button type="button" class="workButtonCard">See Project</button>
                     </div>
-                    <button type="button" class="workButtonCard">See Project</button>
-                </div>
-                <div class="card">
-                    <h2>Multi-Post Stories</h2>
-                    <div class="flexCardTitle">
-                        <h3>CANOPY</h3>
-                        <h4> • Back End Dev • 2015</h4>
-                    </div>
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required</p>
-                    <div class="worksListCard">
-                        <ul class="aligned-list-card">
-                            <li>Html</li>
-                            <li>CSS</li>
-                            <li>JavaScript</li>
-                        </ul>
-                    </div>
-                    <button type="button" class="workButtonCard">See Project</button>
-                </div>
-                <div class="card">
-                    <h2>Tonic</h2>
-                    <div class="flexCardTitle">
-                        <h3>CANOPY</h3>
-                        <h4> • Back End Dev • 2015</h4>
-                    </div>
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required</p>
-                    <div class="worksListCard">
-                        <ul class="aligned-list-card">
-                            <li>Html</li>
-                            <li>CSS</li>
-                            <li>JavaScript</li>
-                        </ul>
-                    </div>
-                    <button type="button" class="workButtonCard">See Project</button>
-                </div>
-                <div class="card">
-                    <h2>Multi-Post Stories</h2>
-                    <div class="flexCardTitle">
-                        <h3>CANOPY</h3>
-                        <h4> • Back End Dev • 2015</h4>
-                    </div>
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required</p>
-                    <div class="worksListCard">
-                        <ul class="aligned-list-card">
-                            <li>Html</li>
-                            <li>CSS</li>
-                            <li>JavaScript</li>
-                        </ul>
-                    </div>
-                    <button type="button" class="workButtonCard">See Project</button>
                 </div>
             </div>
         </section>
-        <section>
+        <section class="aboutMeSecction">
```

Comment:
- [ ] I think it would be great if you increase the size of the shapes next to the texts to match the Figma design. ✍️

**Your design**
![Screenshot from 2023-06-09 07-10-33](https://github.com/alexansaa/Personal_Portafolio/assets/17243325/e7931bd4-5aee-482c-80c6-774370d88ab1)


**Figma design**
![Screenshot from 2023-06-09 07-10-44](https://github.com/alexansaa/Personal_Portafolio/assets/17243325/1282a1de-aa6e-49b2-80dc-e6c016c65377)